tests/lib/agent/Agent.test.ts

```diff
@@ -81,6 +82,9 @@ describe('Agent class', () => {
 
         const decryptedFilePath = await client.decryptFile(nodePath, encryptedFilePath)
         const decryptedData = fs.readFileSync(decryptedFilePath)
+        // console.log(fileContent);
```

Comment:
Don't leave comments in the source code. Turn them into further issues... etc.

Reply:
ahh yes, I've taken these out